### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aritraghosh**
Added AKS enablement of NPD

### CODE REVIEW

**Comment by aritraghosh** [comment]
@dchen1107 @wangzhen127 Can you review

File: README.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ node-problem-detector can either run as a
 [DaemonSet](http://kubernetes.io/docs/admin/daemons/) or run standalone.
 Now it is running as a
 [Kubernetes Addon](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons)
-enabled by default in the GKE cluster.
-
+enabled by default in the GKE cluster. It is also enabled by default in AKS as part of the [AKS Linux Extension](https://learn.microsoft.com/en-us/azure/aks/faq#what-is-the-purpose-of-the-aks-linux-extension-i-see-installed-on-my-linux-vmss-instances).
```
Comment (mmiranda96) [inline README.md:13]: Please split the line into two:
```suggestion
enabled by default in the GKE cluster. It is also enabled by default in AKS as part of the
[AKS Linux Extension](https://learn.microsoft.com/en-us/azure/aks/faq#what-is-the-purpose-of-the-aks-linux-extension-i-see-installed-on-my-linux-vmss-instances).
```

**Review by mmiranda96 — COMMENTED** [review]
/ok-to-test

**Comment by aritraghosh** [comment]
@mmiranda96  done. Please review

**Pull request merged**